apps/common/test/support/code_sigil.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Lexical.Test.CodeSigil do
+  def sigil_q(text, opts \\ []) do
+    ["", first | rest] = text |> String.split("\n")
+    base_indent = indent(first)
+    indent_length = String.length(base_indent)
+
+    [first | rest]
+    |> Enum.map_join("\n", &strip_leading_indent(&1, indent_length))
+    |> maybe_trim(opts)
+  end
+
+  defp maybe_trim(iodata, [?t]) do
+    iodata
+    |> IO.iodata_to_binary()
+    |> String.trim_trailing()
+  end
+
+  defp maybe_trim(iodata, _) do
+    IO.iodata_to_binary(iodata)
+  end
```

Comment:
```suggestion
    [first | rest]
    |> Enum.map_join("\n", &strip_leading_indent(&1, indent_length))
    |> IO.iodata_to_binary()
    |> maybe_trim(opts)
  end

  defp maybe_trim(binary, [?t]) do
    String.trim_trailing(binary)
  end

  defp maybe_trim(binary, _) do
    binary
  end
```
Don't know if there's a particular reason why `iodata_to_binary` was duplicated but `maybe_trim` doing the conversion feels unexpected to me.

Reply:
agreed.